src/Symfony/Component/Yaml/Inline.php

```diff
@@ -591,6 +591,7 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                     case 0 === strpos($scalar, '!!binary '):
                         return self::evaluateBinaryScalar(substr($scalar, 9));
                     case preg_match('/^(-|\+)?[0-9][0-9,]*(\.[0-9_]+)?$/', $scalar):
+                        @trigger_error('Using the comma as a group separator for floats is deprecated since version 3.2 and will be removed in 4.0.', E_USER_DEPRECATED);
```

Comment:
But it should be triggered for `123.45_67` though (I think), as there are no cases for that above this case, and that is valid though. And I think a `_` may have been forgotten in the part before the `.` actually.

IMO, we should do a `strpos` test to test that a `,` is indeed in the match, and then only trigger the deprecation